composer.json

```diff
@@ -15,7 +15,7 @@
         "doctrine/orm": "^2.6",
         "friendsofphp/php-cs-fixer": "^2.14",
         "nesbot/carbon": "*",
-        "phpunit/phpunit": "^5.7",
+        "phpunit/phpunit": "^7.0",
```

Comment:
can we support 8 for php `7.2+`?
i suggest to try: `"phpunit/phpunit": "^7.0 || ^8.0"`

Reply:
It looks like phpunit 8 is causing some troubles. The test suite indeed is exiting with 0, but something else makes it fail on Travis

Reply:
it's not phpunit error. 
we must fix CS